.github/workflows/coverage-linux.yml

```diff
@@ -12,7 +11,6 @@ env:
   FLAKY_TESTS: dontcare
 
 jobs:
-  # TODO(bcoe): add support for C++ coverage.
   coverage-linux:
     runs-on: ubuntu-latest
     steps:
```

Comment:
May want to consider making some of these [conditional](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif), since they may not be helpful on all kinds of PRs (for example, would it be useful on a PR updating GitHub Actions?)

Reply:
(I just ran into an issue where there are a lot of unnecessary Actions CI jobs running on a PR - figured I'd raise it here too, since it seems mildly relevant)

Reply:
+1, same for `doc/`-only changes, `deps/`, `benchmark/`, and `tools/`.

Reply:
I will make these changes.

Reply:
added the paths to ignore 👍